src/v/raft/probe.cc

```diff
@@ -61,108 +61,110 @@ void probe::setup_metrics(const model::ntp& ntp) {
+         sm::description(
+           "Number of replicate requests with leader ack consistency"),
+         labels),
+       sm::make_counter(
+         "replicate_ack_none_requests",
+         [this] { return _replicate_requests_ack_none; },
+         sm::description(
+           "Number of replicate requests with no ack consistency"),
+         labels),
+       sm::make_counter(
+         "done_replicate_requests",
+         [this] { return _replicate_requests_done; },
+         sm::description("Number of finished replicate requests"),
+         labels),
+       sm::make_counter(
+         "log_flushes",
+         [this] { return _log_flushes; },
+         sm::description("Number of log flushes"),
+         labels),
+       sm::make_counter(
+         "log_truncations",
+         [this] { return _log_truncations; },
+         sm::description("Number of log truncations"),
+         labels),
+       sm::make_counter(
+         "leadership_changes",
+         [this] { return _leadership_changes; },
+         sm::description("Number of leadership changes"),
+         labels),
+       sm::make_counter(
+         "replicate_request_errors",
+         [this] { return _replicate_request_error; },
+         sm::description("Number of failed replicate requests"),
+         labels),
+       sm::make_counter(
+         "heartbeat_requests_errors",
+         [this] { return _heartbeat_request_error; },
+         sm::description("Number of failed heartbeat requests"),
+         labels),
+       sm::make_counter(
+         "recovery_requests_errors",
+         [this] { return _recovery_request_error; },
+         sm::description("Number of failed recovery requests"),
+         labels),
+       sm::make_counter(
+         "recovery_requests",
+         [this] { return _recovery_requests; },
+         sm::description("Number of recovery requests"),
+         labels),
+       sm::make_counter(
+         "group_configuration_updates",
+         [this] { return _configuration_updates; },
+         sm::description("Number of raft group configuration updates"),
+         labels),
+       sm::make_counter(
+         "replicate_batch_flush_requests",
+         [this] { return _replicate_batch_flushed; },
+         sm::description("Number of replicate batch flushes"),
+         labels),
+       sm::make_counter(
+         "lightweight_heartbeat_requests",
+         [this] { return _lw_heartbeat_requests; },
+         sm::description("Number of lightweight heartbeats sent by the leader"),
+         labels),
+       sm::make_counter(
+         "full_heartbeat_requests",
+         [this] { return _full_heartbeat_requests; },
+         sm::description("Number of full heartbeats sent by the leader"),
+         labels),
+       sm::make_histogram(
+         "batch_size",
+         [this] { return _batcher_batch_size.internal_histogram_logform(); },
+         sm::description("Batch size from replicate batcher."),
+         labels)},
```

Comment:
only change is this, the rest of it is clang-(re)-formatting.

Reply:
Is reformatting needed? Maybe if we leave braces on their own lines, and add `,` at the end it won't reformat. And as a bonus all metric registrations will be really uniform :)

Reply:
> add `,` at the end

Yeh! This is often the trick!